lib/omnibus/packagers/rpm.rb

```diff
@@ -521,5 +521,29 @@ def safe_architecture
         Ohai['kernel']['machine']
       end
     end
+
+        #
+    # Install the specified packages
+    #
+    # @return [void]
+    #
+    def install(packages, enablerepo)
+      if null?(enablerepo)
```

Comment:
Ok, scratch that, hadn't seen that it's a custom function that's used everywhere :|

The only thing that's probably missing is to have `enablerepo` as an optional parameter of the `extends_packages` DSL method, i.e. have `def extends_packages(packages, enablerepo = NULL)`